package.json

```diff
@@ -14,6 +14,7 @@
     "site:build": "npm run vuepress:build",
     "site:start": "npm run vuepress:start",
     "build": "lerna run build",
+    "generate:types": "node scripts/generate-declarations.js",
```

Comment:
This looks amazing.

I only wish this would be an unscoped package instead of a local script